shopgoodwill.py

```diff
@@ -415,6 +419,39 @@ def get_query_results(
                 ):
                     return total_listings
 
+    def get_item_shipping_estimate(self, item_id: int, zip_code: str) -> float:
+        """
+        Given an item id and a zip code, returns the extracted estimated
+        shipping cost result.
+
+        :param item_id: A valid Shopgoodwill item id
+        :type item_id: int
+        :param zip_code: A valid US zip or zip+4 code
+        :type zip_code: str
+        :return: A float representation of the estimated shipping cost
+        extracted from the xml api response
+        :rtype: float
+        """
+
+        resp = self.shopgoodwill_session.post(
+            f"{Shopgoodwill.API_ROOT}/itemDetail/CalculateShipping",
+            json={
+                "itemId":item_id,
```

Comment:
please lint this branch with `black`

Reply:
Addressed in ace4c3e7

Huh, I hadn't used that before.  I might have to start using that in my day-to-day.  A lot easier than trying to manually stick to a certain code format.